api/src/main/java/org/allaymc/api/form/element/Header.java

```diff
@@ -0,0 +1,29 @@
+package org.allaymc.api.form.element;
+
+import lombok.Getter;
+
+/**
+ * @author daoge_cmd
+ */
+public final class Header implements SimpleFormElement, CustomFormElement {
+
+    @SuppressWarnings("unused")
+    private final String type = "header";
+
+    @Getter
+    private final String text;
+
+    /**
+     * Create a new divider.
```

Comment:
Should be called header instead of divider

Reply:
Oh my fault (((